independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Kiveshan Naidoo"

```

Comment:
@Kiveshan please add 
    "stackOwner": "Name of company or organization that connector is connecting to."